### ISSUE TITLE
Give toggle buttons a unique accessibility label

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#2341]
### Description of problem
All toggle buttons have the same accessibility label (toggle), meaning people who are using voice control, they will not be able to select a particular toggle in a list of toggle options. 

### Expected behavior
Users should be able to toggle a specific option by using the appropriate voice command. Ex. if the user is trying to toggle Developer mode, they should be able to say "tap developer" and have it toggled on/off.

### Steps to reproduce
- enable voice control
- enable dev mode
- go to dev mode

### Environment
- [ ] Occurs on Android
- [x] Occurs on iOS

Build #:

Android Device Model: 

iOS Device Model: 

### Severity
- [ ] High
- [ ] Medium
- [x] Low

**Comment by jeznorth** [comment]
Given the ticket mentions 'all' toggle buttons in the app - do we know what interfaces/components should be updated here?

**Comment by knguyenBC** [comment]
QC team has a ticket in their backlog that helps fix this accessibility ticket. Theirs deal more with adding a selection state but have agreed to make these changes.
